compiler/rustc_resolve/src/lib.rs

```diff
@@ -1686,21 +1686,43 @@ impl<'a> Resolver<'a> {
             .or_insert_with(|| self.arenas.alloc_name_resolution())
     }
 
+    // Test if AmbiguityError ambi is any identical to any one inside ambiguity_errors
```

Comment:
```suggestion
    /// Test if AmbiguityError ambi is any identical to any one inside ambiguity_errors
```

Making this a doc comment will work nicer with IDEs.